src/services/withdrawService.ts

```diff
@@ -347,21 +346,25 @@ async function getBalances(
   }
 }
 
-async function checkNEP141StorageRequirements(
-  {
-    formValues,
-  }: {
-    formValues: WithdrawFormContext
-  },
-  { signal }: { signal: AbortSignal }
-): Promise<
+async function checkNEP141StorageRequirements({
+  formValues,
+}: {
+  formValues: WithdrawFormContext
+}): Promise<
   | { tag: "ok"; value: bigint }
   | { tag: "err"; value: { reason: "ERR_NEP141_STORAGE" } }
 > {
   assert(formValues.parsedRecipient != null, "parsedRecipient is null")
 
+  if (
+    formValues.tokenOut.chainName !== "near" ||
+    isNativeToken(formValues.tokenOut)
```

Comment:
What edge case is covered by isNativeToken function here ?
@cawabunga-bytes

Reply:
<img width="815" alt="Screenshot 2024-12-19 at 13 02 21" src="https://github.com/user-attachments/assets/eba9a0f9-4148-45e2-aad4-b4431772bea2" />


Can this be handled like this ?

Reply:
I would rather have a separate function that accepts a token and return a bool that says if storage deposit is necessary for this token

Reply:
And put there all the edge cases and checks